workshops/food-and-beverage/optimized-distribution/README-Milestone5.md

```diff
@@ -0,0 +1,82 @@
+# Milestone 5 - Enable Event-Driven Notifications
+In this section, you will enable notifications that let the Distributor know a delivery is on its way.  You will then trigger an event by modifying an existing delivery to verify the notifications are sent when the Supplier makes a change.
+
+## Configure Event-Driven Notifications from the DistributorNode
+Modifying the settings of the **DistributorNode** can be done using either the `GraphQL Explorer`, the `Entity Explorer`, or the `Share CLI`.
+
+### Update the DistributorNode's Settings
+You will update the **DistributorNode** to enable email notifications anytime a new block is added to the Vendia Share ledger.  These `blockNotifications` contain information that can be used by the recipient to retrieve additional information, including the transaction submitted that resulted in a new block and even the mutation that was included in the transaction.  For simplicity, you will simply attempt to receieve block notifications from the **DistributorNode**.
+
+* Open the `GraphQL Explorer` of the **DistributorNode**. Remove any existing content from the middle pane.
```

Comment:
This will work, or we could use the web app (Manage Node > Success Notifications).

Reply:
Feel free to add a subsequent section to add another email address via the web app, or just confirm the setting was applied.  Good to show the features of both interfaces.

Reply:
I'll add a section to show how to view and update via the web app. Not blocking for this PR.

Reply:
I added some additional content while I was making updates. Feel free to expand or change.